File: Python/fizz_buzz.py
def fizz_buzz (num):
    fizz = num % 3 == 0
    buzz = num % 5 == 0

    if fizz and buzz:
        print "FizzBuzz"
    elif fizz:
        print "Fizz"
    elif buzz:
        print "Buzz"
    else:
        print num

[fizz_buzz(n) for n in range(1, 101)]

File: PHP/examples/fizzbuzz.php
<?php

/**
* Write a program that prints the numbers from 1 to 100. But for multiples
* of three print “Fizz” instead of the number and for the multiples of five
* print “Buzz”. For numbers which are multiples of both three and five print
* “FizzBuzz”.
**/

function FizzBuzz ($check)
{

    $fizz = $check % 3 == 0;
    $buzz = $check % 5 == 0;

    if ($fizz && $buzz)
    {
        return "FizzBuzz";
    }
    elseif ($fizz)
    {
        return "Fizz";
    }
    elseif ($buzz)
    {
        return "Buzz";
    }

    return $check;
}

for ($index = 1; $index <= 100; $index++)
{
    echo FizzBuzz($index) . "\n";
}

?>
